<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDserver</class>
 <widget class="QMainWindow" name="MDserver">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MDserver</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../build-MDserver-Desktop_Qt_5_14_2_GCC_64bit-Debug/qmake_qmake_immediate.qrc">
    <normaloff>:/icon/server.png</normaloff>:/icon/server.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="server_ip_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>441</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="server_ip_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_ip_line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>441</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="client_ip_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="server_port_line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="client_port_line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>741</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../build-MDserver-Desktop_Qt_5_14_2_GCC_64bit-Debug/qmake_qmake_immediate.qrc"/>
 </resources>
 <connections/>
</ui>
